### DISCUSSION TITLE
Pocketbase in universal load

### DISCUSSION THREAD

**Discussion by SergeyLuashkov** [sveltejs/kit#9537] [Q&A]
I want to use pocketbase in "universal load" but I get the following error:

```
✘ [ERROR] Unexpected "\x7f"

pocketbase:1:0:
  1 │ ELF>@�F@�@8@@@@@@����@�@d...
    ╵ ^

21:49:09 [vite] error while updating dependencies:
Error: Build failed with 1 error:
pocketbase:1:0: ERROR: Unexpected "\x7f"
    at failureErrorWithLog (/run/media/username/workspaces/project/node_modules/esbuild/lib/main.js:1636:15)
    at /run/media/username/workspaces/project/node_modules/esbuild/lib/main.js:1048:25
    at /run/media/username/workspaces/project/node_modules/esbuild/lib/main.js:1512:9
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by CRBroughton** [comment]
I'm having the same issue with Svelte + Typescript and Pocketbase version v0.14.3. I used `pnpm create vite` to bootstrap the project.

Edit: Seems to be related to having the Pocketbase binary in the root dir of the project; Move your binary to a subfolder (db) and run from there.

**Reply by Bouchtaoui** [reply]
Svelte/Vite attempts to add pocketbase in the build process. Moving pocketbase executable outside of the project solves the  problem.
Unless there's a way to tell Vite not to include pocketbase in the build process, this is the only solution.
